app/containers/UIKit/Select.js

```diff
@@ -0,0 +1,87 @@
+import React, { useState } from 'react';
+import { StyleSheet } from 'react-native';
+import PropTypes from 'prop-types';
+import RNPickerSelect from 'react-native-picker-select';
+
+import sharedStyles from '../../views/Styles';
+import { themes } from '../../constants/colors';
+import { CustomIcon } from '../../lib/Icons';
+import { textParser } from './utils';
+import { isAndroid, isIOS } from '../../utils/deviceInfo';
+import ActivityIndicator from '../ActivityIndicator';
+
+const styles = StyleSheet.create({
+	iosPadding: {
+		height: 48,
+		justifyContent: 'center'
+	},
+	viewContainer: {
+		marginVertical: 4,
+		paddingHorizontal: 16,
+		borderWidth: StyleSheet.hairlineWidth,
+		borderRadius: 2,
+		justifyContent: 'center'
+	},
+	pickerText: {
+		...sharedStyles.textRegular,
+		fontSize: 16
+	},
+	icon: {
+		right: 16
+	},
+	loading: {
+		padding: 0
+	}
+});
+
+export const Select = ({
+	options = [],
+	placeholder,
+	onChange,
+	loading,
+	value: values,
+	theme
+}) => {
+	const [initial] = values || [!placeholder];
+	const [selected, setSelected] = useState(initial);
+	const items = options.map(option => ({ label: textParser([option.text]), value: option.value }));
+	const pickerStyle = {
+		...styles.viewContainer,
+		...(isIOS ? styles.iosPadding : {}),
+		borderColor: themes[theme].separatorColor,
+		backgroundColor: themes[theme].backgroundColor
+	};
+
+	const Icon = () => (
+		loading
+			? <ActivityIndicator style={styles.loading} />
+			: <CustomIcon size={22} name='arrow-down' style={isAndroid && styles.icon} color={themes[theme].auxiliaryText} />
+	);
+
+	return (
```

Comment:
It's a common behaviour on iOS, where placeholder is listed as a option, so you can put on placeholder to set it "blank", but on poll app we have placeholder with the same text of the first option. 😅

Reply:
Resolved by https://github.com/RocketChat/Rocket.Chat.ReactNative/pull/1497/commits/51633c726a86f5093d4b20f25811f89008af5e9a.
Now we remove placeholder from options, if has any option selected as initialValue.
Just FYI, if you test it with `/koko question` can cause some confusion, because the select from this app are returning an incorrect initialValue for now.